README.md

```diff
@@ -9,18 +9,12 @@
 
 `p256k1` provides access to curve points in Jacobian coordinates, which allows for very fast operations.  Standard math operators are implemented to allow writing code which closely resembles the underlying math.  The `BitXor` operator is used for scalar exponentiation, using a fast square and multiply algorithm; this provides a very natural and intuitive API.  Fast multi-exponentiation is provided using the standard Pippenger algorithm.
 
-## Dependencies
-Stable rust with a working cargo.  Also need llvm to build secp256k1 and link the wrappers.
+Refer to the [p256k1/](p256k1/) project for more information.
 
-### Ubuntu
```

Comment:
why remove the ubuntu section?

Reply:
oh we don't need llvm anymore?

Reply:
This section is currently in `https://github.com/Trust-Machines/p256k1/tree/sergey/43-download-zip/p256k1`